include/clangmetatool/collectors/definitions_data.h

```diff
@@ -0,0 +1,26 @@
+#ifndef INCLUDED_DEFINITIONS_DATA_H
+#define INCLUDED_DEFINITIONS_DATA_H
+
+#include <string>
+#include <clang/AST/Decl.h>
+
+namespace clangmetatool {
+namespace collectors {
+
+class FileIdHasher {
+public:
+    size_t operator() (const clang::FileID& fileID) const {
+        return fileID.getHashValue();
+    }
+};
+
+class DefinitionsData {
+    public:
+    // filename that contains definition -> AST Node
+    std::unordered_multimap<const clang::FileID, const clang::NamedDecl *, FileIdHasher> defs;
```

Comment:
In the `IncludeGraph` we actually use the `UID` of the file, we even have `clangmetatool::types::FileUID` that we use for that. It's a more friendly thing to use as the key of the map since it is just a number.